README.md

```diff
@@ -87,6 +87,29 @@ client = Mailtrap::Sending::Client.new(api_key: 'your-api-key')
 client.send(mail)
 ```
 
+### ActionMailer
+
+This gem also adds ActionMailer delivery method. To configure it, add following to your ActionMailer configuration (in Rails projects located in `config/$ENVIRONMENT.rb`):
+```ruby
+config.action_mailer.delivery_method = :mailtrap
+config.action_mailer.mailtrap_settings = {
+  api_key: ENV.fetch('MAILTRAP_API_KEY'),
+  api_host: ENV.fetch('MAILTRAP_API_HOST'),
+  api_port: ENV.fetch('MAILTRAP_API_PORT')
```

Comment:
IMO the host and port settings should not be in the default example, as most people should not need to change them.

Reply:
updated, thanks